src/Control/CookieJar.php

```diff
@@ -63,26 +64,31 @@ public function __construct($cookies = array())
      *
      * @param string $name The name of the cookie
      * @param string $value The value for the cookie to hold
-     * @param int $expiry The number of days until expiry; 0 indicates a cookie valid for the current session
+     * @param int $expiry A Unix timestamp of the time the cookie expires; 0 expires at the end of the current session
      * @param string $path The path to save the cookie on (falls back to site base)
      * @param string $domain The domain to make the cookie available on
      * @param boolean $secure Can the cookie only be sent over SSL?
      * @param boolean $httpOnly Prevent the cookie being accessible by JS
      */
-    public function set($name, $value, $expiry = 90, $path = null, $domain = null, $secure = false, $httpOnly = true)
+    public function set($name, $value, $expiry = 0, $path = null, $domain = null, $secure = false, $httpOnly = true)
```

Comment:
I wonder if we could switch to `$expiry = null` and then at the start of the method add:

```php
if ($expiry === null) {
    $expiry = strtotime('+90 days');
}
```

Reply:
`$expiry = null` is currently used as a "clear" flag (i.e `if ($expiry < 0) {`. Is this suggestion also suggesting to alter that convention @kinglozzer ?

Reply:
There are existing cases in framework where `null` is used to "clear" the cookie (i.e expire when the session expires).

https://github.com/silverstripe/silverstripe-framework/blob/4/src/Security/MemberAuthenticator/CookieAuthenticationHandler.php#L265-L272

So establishing `null` as "fallback to default" is incongruent with existing behaviour, and it's uncertain if other users have replicated this as a convention.

Do you think that's prohibitive?